tools/syz-trace2syz/config/unsupported_calls.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package config
+
+var (
+	// ShouldSkip lists system calls that we should skip when parsing
+	// Some of these are unsupported or not worth executing
+	ShouldSkip = map[string]bool{
+		// execve terminates program execution. Although there is Syzkaller support
```

Comment:
1. We never spell syzkaller with a capital letter.
2. I've noticed here and in some other places that you refer to syzkaller in third person form. It made sense while your code was a separate project, but this is a part of syzkaller now. So you should refer to it in first person form. It's not _there_ now, it's _here_. This code is also part of syzkaller now. So referring to "some syzkaller" is not useful anymore. You could say something along the following lines "Although syscall descriptions contain execve ...".